common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationRequest.java

```diff
@@ -72,12 +83,17 @@
      * encode information about the user's state in the app before the authentication request
      * occurred, such as the page or view they were on.
      */
+    @SerializedName("state")
     private String mState;
 
+    //Marking as transient to avoid these values being serialized by GSON or other Java Serialization
+    private transient Activity mActivity;
```

Comment:
Why we need these two properties in the authorization request? Shouldn't it belong to the authorization strategy where handling the UI flow?

Reply:
I tend to agree.  @iambmelt gave the same feedback.  I'll take a look.